docs/ossm/ossm2-migration/cluster-wide/README.md

```diff
@@ -66,41 +65,43 @@ In this procedure it's expected that all 2.6 data plane namespaces have `istio-i
 1. Prepare the `Istio` resource yaml named `ossm-3.yaml` to be deployed to the same namespace as the 2.6 control plane:
 
     Here we are not using any `discoverySelectors` so the control plane will have access to all namespaces. In case you want to define `discoverySelectors`, keep in mind that all data plane namespaces you are planning to migrate from 2.6 must be matched.
-    
-    Also note that `default` name with `InPlace` update strategy is used which allows usage of the `istio-injection=enabled` label. In case you want to use different name or `RevisionBased` update strategy, you will have to configure `default` `IstioRevisionTag`.
 
     ```yaml
    apiVersion: sailoperator.io/v1alpha1
    kind: Istio
    metadata:
-     name: default # the name and the updateStrategy is significant for injection labels
+     name: ossm-3 # the name, updateStrategy and version are significant for injection labels
    spec:
      updateStrategy:
-       type: InPlace # the name and the updateStrategy is significant for injection labels
-     namespace: istio-system # the same namespace where we run the 2.6 control plane
+       type: RevisionBased
+     namespace: istio-system # 3.0 and 2.6 control planes must run in the same namespace
      version: v1.24.1
    ```
 1. Apply the `Istio` resource yaml:
 
-    > **_NOTE:_** after next step, both 2.6 and 3.0 control planes will try to inject side cars to all pods in namespaces with the `istio-injection=enabled` label and all pods with the `sidecar.istio.io/inject="true"` label after next restart of the workloads. Workloads should be restarted only after the `maistra.io/ignore-namespace: "true"` label is added (see below).
     ```sh
     oc apply -f ossm-3.yaml
     ```
 1. Verify that new `istiod` is using existing root certificate:
 
     ```sh
-    oc logs deployments/istiod -n istio-system | grep 'Load signing key and cert from existing secret'
+    oc logs deployments/istiod-ossm-3-v1-24-1 -n istio-system | grep 'Load signing key and cert from existing secret'
     2024-12-18T08:13:53.788959Z	info	pkica	Load signing key and cert from existing secret istio-system/istio-ca-secret
     ```
+
 ##### Migrate Workloads
-1. Add `maistra.io/ignore-namespace: "true"` label to the data plane namespace
+This procedure is not using revision tags but it's recommended to use them for big meshes to avoid re-labeling of namespaces during future 3.y.z upgrades.
```

Comment:
Also if they want to keep using `istio-injection: enabled` label after the migration they'll need to create a revision tag.

Reply:
Nevermind this is covered below by the next scenario. Is the only difference between the first two scenarios the creation of the istio rev tag after? Maybe we can consolidate the two scenarios then and have an optional section if you want to keep using `istio-injection: enabled` where the istio rev tag is created?